### PR TITLE
Bug 1986302: Check rbac before polling for rules silences

### PR DESCRIPTION
**Pull request by jcaianirh**
Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=1986302

### CODE REVIEW

File: frontend/public/components/notification-drawer.tsx [hunk 1]
```diff
@@ -193,63 +193,73 @@ export const ConnectedNotificationDrawer_: React.FC<ConnectedNotificationDrawerP
...
   React.useEffect(() => {
-    const poll: NotificationPoll = (url, key: 'notificationAlerts' | 'silences', dataHandler) => {
-      dispatch(alertingLoading(key));
-      const notificationPoller = (): void => {
-        coFetchJSON(url)
-          .then((response) => dataHandler(response))
-          .then((data) => dispatch(alertingLoaded(key, data)))
-          .catch((e) => dispatch(alertingErrored(key, e)))
-          .then(() => (pollerTimeouts[key] = setTimeout(notificationPoller, 15 * 1000)));
+    if (rulesAccess && isRulesAccessLoading) {
```
Comment (jhadvig) [inline frontend/public/components/notification-drawer.tsx:202]: Dont think this is actually right. Shouldn't the `isRulesAccessLoading` be actually set to `false` to proceed ?

Comment (jcaianirh) [reply frontend/public/components/notification-drawer.tsx]: @jhadvig removed the isloading flag because it wasn't necessary for the check.

**Review by jhadvig — APPROVED** [review]
/lgtm
/approve

**Pull request merged**